### ISSUE TITLE
grafana-cli plugins can't update to 2.0.2

### ISSUE THREAD

**Issue by Stephane-Thales** [Vertamedia/clickhouse-grafana#244]
Dear All,

I've the pluggin 2.0.1, trying to upgrade to 2.0.2
with Grafana v7.1.0 (8101355285)

using
`grafana-cli plugins update vertamedia-clickhouse-datasource`
or 
`grafana-cli plugins update-all`

the cmd line keep doing the upgrade, but in the UI the plugin stay in 2.0.1, and if I rerun the command it's upgrading it again, without doing anything.
I've "upgraded" the plugin multuple time, restarted grafana service, rebooted the VM, no change.

Any idea where I can find clues of what is not happening ?

Thanks

**Comment by Slach** [comment]
duplicate with https://github.com/Vertamedia/clickhouse-grafana/issues/243

**Comment by Slach** [comment]
@Stephane-Thales could you run following commands?
```bash
systemctl stop grafana-server
grafana-cli update-all
systemctl start grafana-server
```

look like backend part of plugin /var/lib/grafana/plugins/vertamedia-clickhouse-grafana/dist/vertamedia-clickhouse-plugin_linux_amd64`  is locked when grafana-server is running

**Comment by Stephane-Thales** [comment]
Hi @Slach 

That did not work.

I also tried without success

```
systemctl stop grafana-server
grafana-cli plugins uninstall vertamedia-clickhouse-datasource
grafana-cli plugins install vertamedia-clickhouse-datasource
systemctl start grafana-server
```

But what worked :

```
systemctl stop grafana-server
grafana-cli plugins uninstall vertamedia-clickhouse-datasource
cd /var/lib/grafana/plugins
rm -rf clickhouse-grafana
grafana-cli plugins install vertamedia-clickhouse-datasource
systemctl start grafana-server
```

And that upgrade to 2.0.2 fixed my issue #199 .
Thanks

**Comment by Slach** [comment]
hmmm look like you install clickhouse grafana plugin into
`/var/lib/grafana/plugins/clickhouse-grafana` manually

but `grafana-cli plugins install` use `/var/lib/grafana/plugins/vertamedia-clickhouse-datasource`
